ModelCatalogueCorePluginTestApp/grails-app/conf/Config.groovy

```diff
@@ -201,24 +201,28 @@ environments {
         // ---
         grails.logging.jul.usebridge = false
         println "ServerURL:" + grails.serverURL
-        println "MDX_SERVER_URL:" + System.getenv('MDX_SERVER_URL')
-        grails.serverURL = System.getenv('MDX_SERVER_URL')//"https://localhost:8899/mc"
+        println "MC_SERVER_URL:" + System.getenv('MC_SERVER_URL')
+        grails.serverURL = System.getenv('MC_SERVER_URL')//"https://localhost:8899/mc"
         //println "ServerURL:" + grails.serverURL
         println "ServerURL:${grails.serverURL} END"
-        println "MDX_DB_URL:" + System.getenv('MDX_DB_URL')
```

Comment:
Why have all of these changed - will this mess up the config. i.e. shouldn't it be MDX_DB_URL not MC_DB_URL